model/src/main/java/com/expediagroup/streamplatform/streamregistry/model/ConsumerBinding.java

```diff
@@ -36,7 +40,5 @@
   @EmbeddedId
   private ConsumerBindingKey key;
   private Specification specification;
-  private String consumerId; //Resolves to Consumer
-  private String bindingId; //Resolves to StreamBinding
```

Comment:
Are you sure these can come out?

Reply:
They're not there on the `ProducerBindingKey` and that works, and the build tests pass 👍

Reply:
ok